conversion/conversion.go

```diff
@@ -129,12 +130,15 @@ func connectionConfig(driver string, sqlConnectionStr string) (interface{}, erro
 		}
 		return sqlConnectionStr, nil
 	case constants.MYSQL:
-		// If empty, this is called as part of the legacy mode witih global CLI flags.
-		// When using source-profile mode is used, the sqlConnectionStr is already populated.
 		if sqlConnectionStr == "" {
 			return generateMYSQLConnectionStr()
 		}
 		return sqlConnectionStr, nil
+	case constants.SQLSERVER:
+		if sqlConnectionStr == "" {
+			return generateSQLSERVERConnectionStr()
```

Comment:
Since we don't support the env variables anymore, sqlConnectionStr should never be empty. Consider refactoring this and adding a comment on why its left empty.

Reply:
Agreed, updated the code accordingly. Changes should be available now.